client/ped.lua

```diff
@@ -33,6 +33,9 @@ function cAPI.SetPedScale(ped, num)
 end
 
 function cAPI.SetPedOverlay(ped, data)
+    if data == nil then
```

Comment:
Change the "if data == nil then" to "if not data then"

Reply:
This checks whether the data is "nill". If we use "not data" there is still a possibility of getting an error. Because in case data = false, this will return true and an error will still occur.
"not data" does not equal nil

Reply:
![image](https://github.com/Faroeste-Roleplay/frp_core/assets/20740173/564d87bc-8731-403a-8700-f69af7e0ae27)